data-restore.env

```diff
@@ -15,4 +15,24 @@ PG_DUMP=
 # If set to false, the driver will continuously scan Ethereum blocks unless
 # it is manually interrupted.
 # true (default if left empty) or false
-FINITE_MODE=
+FINITE_MODE=false
+
+# The next parameters are required to run a local node and not relevant for 
+# the data-restore tool
+
+# The URL of the Postgres database bootstrapped by the docker-compose
+DATABASE_URL=postgres://postgres@postgres:5432/plasma
+# The next three parameters aren't relevant for the local node, but needed to be supplied anyway
+DATABASE_POOL_SIZE=10
+DATABASE_REJECTED_TRANSACTIONS_MAX_AGE=336
+DATABASE_REJECTED_TRANSACTIONS_CLEANER_INTERVAL=24
+
+# The port to which the zksync node will bind to
+API_WEB3_PORT=3002
+# Max difference between blocks in `eth_getLogs` method.
+API_WEB3_MAX_BLOCK_RANGE=10
```

Comment:
It's safe to make this number bigger, especially for local usage

Reply:
I will leave a comment about it